### ISSUE TITLE
Disable voice input and QR Code reader

### ISSUE THREAD

**Issue by n8fr8** [amoghbl1/tor-browser#19]
Settings->Accessibility->Voice input and QR code reader should be disabled by default (though users who need them can enable them!)
